shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -191,12 +191,6 @@ function shoov_ci_build_create_build_item($node, $timestamp = NULL) {
  */
 function shoov_ci_build_message_update(Message $message) {
   $wrapper = entity_metadata_wrapper('message', $message);
-  $status = $wrapper->field_ci_build_status->value();
```

Comment:
this is wrong, by removing it - you might create duplicate queued items. You probably just need to move the `shoov_ci_build_trigger_event($message, 'ci_build_update');` above this, no?